bin/_docker.sh

```diff
@@ -33,6 +33,7 @@ export DOCKER_IMAGES=(${DOCKER_IMAGES:-
     cni-plugin
     controller
     policy-controller
+    linkerd-cni-validation
```

Comment:
i think we renamed this to
```suggestion
    linkerd-cni-validator
```